docs/docs/contributing.md

```diff
@@ -0,0 +1 @@
+../../CONTRIBUTING.md
```

Comment:
I created a symlink on my machine with `ln -s ../../CONTRIBUTING.md docs/docs/contributing.md` but idk if that only works locally?

Reply:
git respects symlinks